examples/closed-issue-counter/job.bal

```
@@ -0,0 +1,87 @@
+
+    public function execute() {
+        log:printInfo("Job started to executed.");
+        time:Utc date = time:utcNow();
+        time:Civil civil = time:utcToCivil(date);
+        string months = civil.month.toString();
+        if months.length() == 1 {
+            months = "0" + months;
+        }
+        string days = civil.day.toString();
+        if days.length() == 1 {
+            days = "0" + days;
+        }
+        string dateInString = (civil.year).toString() + "-" + months + "-" + days;
+        string|error closedIssueCounts = getClosedIssueCounts(repoNamesWithLabel, dateInString);
+        if closedIssueCounts is string {
+            log:printInfo("Closed issue count details on " + dateInString + ":\n" + closedIssueCounts);
+        } else {
+            log:printError("Failed to get the closed issue count details", 'error = closedIssueCounts);
+        }
+    }
+
+    isolated function init() {}
+}
+
+function getClosedIssueCounts(string[] entries, string date) returns string {
+    string totalCount = "";
+    foreach string entry in entries {
+        string[] orgname = regex:split(entry, ":");
+        json|error issueCount;
+        if orgname.length() > 1 {
+            issueCount = githubClient->getRepositoryIssueList(repositoryOwner, orgname[0], date, orgname[1]);
+        } else {
+            issueCount = githubClient->getRepositoryIssueList(repositoryOwner, orgname[0], date);
+        }
+        if issueCount is json {
+            json|error value = issueCount.data.search.issueCount;
+            if value is json {
+                if orgname.length() > 1 {
+                    totalCount += orgname[0] + ":" + orgname[1] + " " + value.toString() + "\n";
+                } else {
+                    totalCount += orgname[0] + " " + value.toString() + "\n";
+                }
+            } else {
+                log:printError("Error:", value);
+            }
+        } else {
+            log:printError("Error:", issueCount);
```

shall we return the error by `check` and handle it inside execute function